bootstrap.sh

```diff
@@ -2,7 +2,7 @@
 export DEBIAN_FRONTEND=noninteractive
 echo "Updating Packages & Installing Dependencies"
 apt-get update
-apt-get install -y git-core automake pkg-config libtool m4 libbsd-dev libssl-dev libmilter-dev emacs python3-pip libyaml-dev
```

Comment:
Having emacs in there just adds time to provisioning and if people want it they can add it once they ssh in. Just my .02